tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -383,7 +383,7 @@ def expr_eval(expr):
                 else:
                     print('NO MATCH', verbose_pattern(topic, msg, t))          
 
-                total_bytes += len(serialized_bytes)
+                total_bytes += len(serialized_bytes) 
```

Comment:
```suggestion
                total_bytes += len(serialized_bytes)
```

Reply:
The whitespace errors are, weirdly, intentional. Upstream ros_comm has a strict "no unrelated whitespace changes" policy, so on the off chance they'll accept this without tests I added these back in.

Reply:
Fair enough. We should merge it in to here too though: https://github.com/locusrobotics/rosbag_snapshot

Reply:
Just so I'm sure we're on the same page: "We should merge it in here too" = we should add an option to `snapshot` to retain latched messages?

Reply:
You're right...I was thinking this was a part of the snapshot logic.